R/return_linter.R

```diff
@@ -130,6 +141,12 @@ return_linter <- function(
 
   Linter(linter_level = "expression", function(source_expression) {
     xml <- source_expression$xml_parsed_content
+    if (check_except && !is.null(except_regex)) {
```

Comment:
Is it possible to do static exclusion for `except` and post-process resulting lints to filter out regex matches? This approach is hard for me to understand due to the many branches.

Reply:
Hmm actually I think it may be hard due to the recursion.

```r
ExcludeMe <- function() {
  lapply(seq_len(10), \(i) {
    i_need_a_return()
  })
  i_dont()
}
```

Not sure how to handle this without adding an unwind step for the recursion which is probably too much.